app/views/layouts/application.html.erb

```diff
@@ -80,8 +79,20 @@
       </div>
     </div>
   </header>
-
+  
```

Comment:
Is there meant to be a whitespace there? I think you can configure your editor to automatically get rid of trailing whitespace.